uploader/package-updater.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+const filename = path.resolve(path.join(__dirname, "../package.json"));
+const encoding = { encoding: "utf8" };
+
+module.exports = function(driverVersion, nwVersion, hash) {
+  return readPackageJson()
+    .then(function(json) {
+      json.version = driverVersion;
+      json.versionHash = hash;
+      json.engines.nw = nwVersion;
```

Comment:
Not sure `engines` is the best place to put this metadata. It implies the module only works with that version, which isn't really true. Indeed, you're not really even executing the module at all, if I understand correctly. It also doesn't communicate "this came from" rather than "this works with" which are different. I think putting it in a place that can implicitly communicate that would be better.